reduction.py

```diff
@@ -104,13 +104,44 @@ def task2(img_df):
     if clustering.n_features_in_ == 0:
   	consensus_flag = 0
 
-    return center_list, consensus_flag, 
-
- 
-# task 3
-def task3(img_df):
-    #dostuff
-    return consensus, consensus_reached, aux_info
+    return center_list, consensus_flag, clustering.labels_, clustering.n_features_in_
```

Comment:
keep the output consistent and  return three objects, i.e.:
maybe change to: center_list, census_flag, [clustering.labels_ , clustering.n_features_in_]

Add to the function description what outputs it includes.